region/brinstar/blue/Morph Ball Room.json

```diff
@@ -90,7 +90,7 @@
     },
     {
       "id": 6,
-      "name": "Morph Ball Room Junction (Right of Sidehoppers)",
+      "name": "Junction (Right of Sidehoppers)",
```

Comment:
`Right of Sidehoppers` isn't clear to me.
I'd like `Left of Morph Tunnel` more.